perk-store-services/src/main/resources/locale/addon/PerkStore_en.properties

```diff
@@ -232,6 +232,8 @@ exoplatform.perkstore.label.profile.members=Members
 
 exoplatform.perkstore.label.openInWallet=Open in wallet
 exoplatform.perkstore.label.addNewProduct=Add New Product
+exoplatform.perkstore.label.SoldOut=Sold Out
+exoplatform.perkstore.label.disabledProduct=Disabled Product
```

Comment:
use the keys that already exist and change it in Crowdin

`exoplatform.perkstore.label.productFiltersDisabledProducts`
`exoplatform.perkstore.label.productFiltersSoldOutProducts`

Reply:
already used can't change them!